### DISCUSSION TITLE
I've made quasar2 + firebase-auth boilerplate

### DISCUSSION THREAD

**Discussion by danbars** [quasarframework/quasar#10726] [Show and Tell]
I've found that wiring authentication to a new project is very time consuming.
I couldn't find any quasar2 + firebase boilerplate, so I've made one: https://github.com/datamule-io/quasar2-vue3-firebase-auth
It's almost vanilla quasar 2 app, with FIrebase authentication including sign-in / sign-up, forgot-password, etc, already wired.
Feel free to use and contribute.
![image](https://user-images.githubusercontent.com/6115771/133880022-05655bf1-8aaf-4488-8e5e-202a07291c8a.png)

**Comment by ontwikkelfabriek** [comment]
@danbars 

Great contribution! A lot of people will be very helped with this starter kit.

**Comment by CynthiaBlue** [comment]
This is awesome, thanks!

**Comment by TobyMosque** [comment]
As a suggestion, u can transform your starter kit in an UI extension. Just put your components, boots, etc in the template folder and render it in the install script.

https://quasar.dev/app-extensions/development-guide/install-api#api-render

If u wanna to learn more why that approach is prefered over create a starter kit, read that:

https://quasar.dev/app-extensions/tips-and-tricks/starter-kit-equivalent#introduction

**Reply by danbars** [reply]
@TobyMosque this is a good idea, I wasn't aware of this option.
Thanks!
